### PR TITLE
[WIP] testing/yq: new aport

### PR DESCRIPTION
**Pull request by dimon222**
yq package installation (pre-compiled binary)

yq is a portable command-line YAML processor 
tl;dr - YAML version of parser like jq

### CODE REVIEW

File: testing/yq/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=yq
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="yq is a portable command-line YAML processor"
+url="https://github.com/mikefarah/yq"
+arch="noarch"
+license="MIT"
+options="!check"
```
Comment (andypost) [inline testing/yq/APKBUILD]: it needs comment explaining reason of missing tests

**Comment by andypost** [comment]
Please squash commits to one

**Comment by andypost** [comment]
Also I find strange that go does not involved in build

**Comment by dimon222** [comment]
Yep, its currently precompiled binary, and tests are against tarball. I'll see if I can run complete compilation with Go, this PR will change as its still WIP

**Pull request closed without merging**